training/culture-and-process/70_20_10_TIME.md

```diff
@@ -0,0 +1,44 @@
+70% 20% 10% Time in DP
+===========================
+
+As part of working in Digital Publishing, the development team share sprint time between feature work (70%) driven by product managers,
+technical workflows and tools (20%) driven by Technical Leads and development teams and self training (10%) driven by a developer.
+
+### Pre-reading
+
+- It is useful to understand [Trello Board - Flow of Stories](./TRELLO_BOARD_FLOW.md) and how we use it to organise development workflow as it is mentioned in this documentation
+
+### 70% Time
+
+70% time work includes:
+- Product features
+- Project (epics)
+- High Priority Bugs and Security Maintenance of applications/infrastructure
+
+### 20% Time
+
+20% time is generally used to work on stories that will improve our working experience within digital publishing.
```

Comment:
this is specific to developers - so either 'the developer experience' or 'our working experience as developers' might be clearer (just in case someone in another role is having a look at any of our culture docs)